src/language-js/print/ternary-old.js

```diff
@@ -283,7 +283,9 @@ function printTernaryOld(path, options, print) {
       consequentNode.type === node.type ? ifBreak("", ")") : "",
       line,
       ": ",
-      align(2, print(alternateNodePropertyName)),
+      options.useTabs
+        ? indent(print(alternateNodePropertyName))
+        : align(2, print(alternateNodePropertyName)),
```

Comment:
```suggestion
      indent(print(alternateNodePropertyName))
```

Will this work?

Reply:
We also have `tabWidth` option, hardcoded `2` seems not correct.

Reply:
I thought that this will reduce the overall width of the conditional expression, probably it was introduced in this PR https://github.com/prettier/prettier/pull/1822

Reply:
The diff with `indent` instead of `align(2, ...)` looks quite frightening https://github.com/auvred/prettier/commit/237a598fb5d4f1a9524337c2f405a53ad446cd30

Should I push it to this PR?

Reply:
You are right, it's bad. But we need add a comment to explain that `2` is not the `tabWidth`, it's actually used to align with `? ` or `: `.

Reply:
Done